DOC-538 Table Collection Docs (#629)
Macrometacorp/docs: docs/cep/enrich-data/enrich-data-with-collection.md

#### Define the Table (Collection)

> `CREATE TABLE` defines where the stream worker stores your data. In this case, it will be a [Document Store Collection](../../collections/documents/index.md) For more information about defining a TABLE in a stream worker, refer to [Table (Collection)](../sink/sink-types/table/index.md). For more information about collections in general, refer to [Collections](../../collections/index.md).

Reviewer: ```suggestion
`CREATE TABLE` defines where the stream worker stores your data. In this case, it is a [document store collection](../../collections/documents/index.md). Refer to the [Table (Collection)](../sink/sink-types/table/index.md) section or information about defining a TABLE in a stream worker, or the [Collections](../../collections/index.md) section or more information about collections in general.
```

Author: I prefer the construction of "To get thing, then do this" in instructions rather than "Do this, then this happens." It tells the user what to expect before they are told to perform an action. In this case, "To get info, click this link."
